src/libraries/System.Net.Sockets/src/Resources/Strings.resx

```diff
@@ -193,7 +194,7 @@
     <value>The specified value cannot be negative.</value>
   </data>
   <data name="ArgumentOutOfRange_Bounds_Lower_Upper" xml:space="preserve">
-    <value>Argument must be between {0} and {1}.</value>
+    <value>Argument {2} must be between {0} and {1}.</value>
```

Comment:
Nit, we're definitely not consistent, but we generally should quote any strings inserted into resources. Single quotes seem to be almost always used.